static/js/message_edit.js

```diff
@@ -118,6 +118,7 @@ exports.save = function (row, from_topic_edited_only) {
         success: function () {
             if (msg_list === current_msg_list) {
                 row.find(".edit_error").text(i18n.t("Message successfully edited!")).removeClass("alert-error").addClass("alert-success").show();
+                row.find(".message_edit").hide(150, "linear");
```

Comment:
I couldn't try out this change yet I guess so if it relates then we should update the speed here as well.